pkg/tls/certificatemanagement/certificatebundle.go

```diff
@@ -99,6 +99,10 @@ func createTrustedBundle(includeSystemBundle bool, name string, ca CertificateIn
 	return bundle, err
 }
 
+func (t *trustedBundle) SetName(name string) {
```

Comment:
I'd rather that we follow the established pattern and have the existing controller (core_controller in this case) add the certificates.

Reply:
Yeah, let's remove this function for now

Reply:
All gone!